src/crypto-api.ts

```diff
@@ -18,6 +18,7 @@ import type { IMegolmSessionData } from "./@types/crypto";
 import { Room } from "./models/room";
 import { DeviceMap } from "./models/device";
 import { UIAuthCallback } from "./interactive-auth";
+import { IImportRoomKeysOpts } from "./crypto/api";
```

Comment:
I'm trying to move things out of `crypto` at the same time as adding them to CryptoApi. Could you move `IImportRoomKeysOpts` into this file and rename it to `ImportRoomKeysOpts`?

Reply:
So:
- `crypto/` is js crypto
- `rust-crypto/` is rust crypto
- and `common-crypto` is the API

Right? So I should move IImportRoomKeysOpts to common ?

Reply:
oh i was confused with crypto/api.ts and crypto-api.ts

Reply:
Updated